### DISCUSSION TITLE
[Bug] Can't move assets after search [Portal-Engine]

### DISCUSSION THREAD

**Discussion by wennstrom** [pimcore/pimcore#14484] [Q&A / Support] [answered]
I found a bug when i try to move assets after a search. 

Here i have searched for and selected assets i want to edit

![Skärmavbild 2023-02-28 kl  12 03 18](https://user-images.githubusercontent.com/54888925/221835979-755da4e6-050f-444f-8975-d9fce235fbb8.png)

And when i click on move nothing shows up in the move modal

![Skärmavbild 2023-02-28 kl  12 03 48](https://user-images.githubusercontent.com/54888925/221837864-3889af1e-485d-47d2-a51f-798bdd21d3b0.png)

It appears to be because parentPath is null in FolderTreeRelocate.js 
Works fine when i move assets without a search

**Comment by fashxp** [comment, marked as answer]
Thx for reporting. We created an [issue](https://github.com/pimcore/portal-engine/issues/345) in the corresponding enterprise repository (only accessible for enterprise customers) and will take a look at it.
